package-lock.json

```diff
@@ -12713,6 +12774,29 @@
         "node": ">=8"
       }
     },
+    "node_modules/katex": {
```

Comment:
OK, I see where the `katex` dependency is coming from, but since it's a dev-time dependency only now (along with everything that's been added) it's also of vastly less concern.